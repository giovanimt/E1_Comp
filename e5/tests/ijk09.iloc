addI rsp, 4 => rsp
addI rsp, 4 => rsp
addI rsp, 4 => rsp
loadI 15 => r0
addI rfp, 4 => r1
store r0 => r1
loadI 3 => r2
addI rfp, 8 => r3
store r2 => r3
addI rfp, 4 => r4
load r4 => r5
addI rfp, 8 => r6
load r6 => r7
add r5, r7 => r8
addI rfp, 4 => r9
load r9 => r10
add r8, r10 => r11
addI rfp, 8 => r12
load r12 => r13
add r11, r13 => r14
loadI 10 => r15
add r14, r15 => r16
addI rfp, 0 => r17
store r16 => r17
