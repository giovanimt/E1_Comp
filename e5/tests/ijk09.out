00022905 46
00022909 15
00022913 3
